### ISSUE TITLE
mindswapping into mindshielded bodies as headrev

### ISSUE THREAD

**Issue by BrumalLake** [BeeStation/BeeStation-Hornet#5139]
## Reproduction:
be a headrev and swap your consciousness into a mindshielded duplicate
you get to be a headrev with a mindshield
probably has other weird interactions like not deconverting revs on bodyswapping into a mindshielded body
tldr remove/replace xenobiology causes more problems than worth
probably happens with the mindswap spell which would be unlikely to happen but possible on dynamicbus

**Comment by TeomanTheGreat** [comment]
hi this was me who accidentally discovered this. So I decided to make myself a second body with xenobio slimeperson stuff so I could send out that second body to kill the cap who according to sensors was in medbay. I think I hit the cmo a few times and converted someone else. Anywho that body was captured and I returned to my original. A little while later I noticed my second body was still alive so I transferred to it. Since I was released I assumed I was mindshielded and a headrev at the same time which I was. 


Image after I killed and replaced HoS. 
![image](https://user-images.githubusercontent.com/55861563/129638476-a99c1077-cd06-4f2a-8004-df832e95ca67.png)
My guess for the reasoning of this is because since my mind was not in the second body I didn't count as a rev as evident as the lack of logo on the rev hud when I was in a different body. I believe a check should be added if possible to extra bodies made by xenobio slime species.
